src/main/kotlin/racingcar/view/input/Input.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar.view.input
+
+open class Input {
+
+    open fun readCar(): Int {
+        return readLine()!!.toInt()
```

Comment:
코틀린 1.6부터 `readLine()!!` 대신 `readln()`을 사용할 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/whatsnew16.html#new-readline-functions

```suggestion
        return readln().toInt()
```

Reply:
오 네 감사합니다! @wotjd243 
(여담인데, 과제 문서 하단에 있는 코드도 같이 수정되면 좋을것 같습니다!)

![image](https://user-images.githubusercontent.com/7760496/168307965-269f427a-262d-4deb-95f0-725984fdd187.png)